src/main/java/org/terasology/dynamicCities/region/components/RegionEntitiesComponent.java

```diff
@@ -28,10 +29,10 @@
 @MappedContainer
 public class RegionEntitiesComponent implements Component {
 
-    public Map<String, EntityRef> regionEntities;
+    public Map<Vector2i, EntityRef> regionEntities;
```

Comment:
Just to be sure, we can make those `Vector2ic` to guard them against accidental mutation.